warehouse/api/http.go

```diff
@@ -70,7 +73,7 @@ type Api struct {
 	notifier      *notifier.Notifier
 	bcConfig      backendconfig.BackendConfig
 	tenantManager *multitenant.Manager
-	bcManager     *backendConfigManager
+	bcManager     *bcm.BackendConfigManager
```

Comment:
We could have an interface here as well. Two methods only are needed.

Reply:
I will add this to the Backlog. I also want to have a unified constructor for `api` package which will have `startHTTPServer` and `startGRPCServer` instead of having separate constructors for HTTP and GRPC.